src/metabase/models/card.clj

```diff
@@ -317,12 +317,29 @@
         (when-not (= parameters new-parameters)
           (db/update! model po-id {:parameters new-parameters}))))))
 
+(defn model-supports-implicit-actions?
+  "A model with implicit action supported iff they are a raw table,
+  meaning there are no clauses such as(filter, limit, breakout...)"
+  [{dataset-query :dataset_query :as _card}]
+  (and (= :query (:type dataset-query))
+       (every? #(nil? (get-in dataset-query [:query %])) [:expressions :filter :limit :breakout :aggregation :joins])))
```

Comment:
it matches the check on FE: https://github.com/metabase/metabase/blob/89f8d515fe544cf85ed810b0b4cdc112222c71b5/frontend/src/metabase-lib/queries/StructuredQuery.ts#L508

Reply:
Again, this kind of comment should live in the code. Whenever there's a co-dependency between BE and FE there should be a comment in both spots, so if the FE updates they know the BE needs to update as well, and vice versa.

Reply:
This also doesn't match the FE perfectly. You need to add `:order-by` and `:fields`
```suggestion
       (every? #(nil? (get-in dataset-query [:query %])) [:expressions :filter :limit :breakout :aggregation :joins :order-by :fields])))
```

Reply:
nice catch, Added the comment and include the missing clauses here: https://github.com/metabase/metabase/pull/29021/commits/1d8c6582972870b47b21603782d8618639ef371d